### DISCUSSION TITLE
Lista de PSPs com API de envio de Pix

### DISCUSSION THREAD

**Discussion by rubenskuhl** [bacen/pix-api#480] [General]
Atualização: 12.09.2024

No #76 eu tenho mantido atualizada uma lista com APIs de recebimento de Pix; este tópico é para lista de envio de Pix por API. Lembrando que não há padrão BACEN para esse caso de uso. Para essa lista apenas os que tem capacidade de envio para chave Pix estão qualificados. Notar que isso antes de 02.09.2024 podia ser implementado por 2 chamadas (uma para obter dados bancários, outra para enviar), mas que desde então somente o destino especificado por chave Pix é possível. 

PSPs com API de envio de Pix (Empresa - Site ou Link API)

- Acesso Soluções de Pagamento S.A. 13.140.088/0001-99 - https://bankly.com.br - https://docs.bankly.com.br/docs/pix-cash-out https://docs.bankly.com.br/docs/pix-consulta-por-chaves
- BANCO BS2 S.A. 71.027.866/0001-34 - https://devs.bs2.com/manual/pix-clientes/
- Banco Inter 00.416.968/0001-01 - https://developers.inter.co/references/banking#tag/Pix-Pagamento
- Banco Rendimento S.A. 68.900.810/0001-38 - https://www.rendimento.com.br/pix-correntistas/
- Dock Instituição de Pagamento S.A. 13.370.835/0001-85 - https://bpp.com.br/pix-pagamentos-instantaneos/
- BTG Pactual 30.306.294/0001-45 - https://developer.btgpactual.com/docpm https://developer.btgpactual.com/docpi
- Ewally Tecnologia e Serviços S.A. 00.714.671/0001-14 - https://www.ewally.com.br/solucoes/baas-banking-as-a-service/
- Efí Pay (antiga Gerencianet) 09.089.356/0001-18 - https://dev.efipay.com.br/docs/api-pix/gestao-de-pix#requisitar-envio-de-pix
- ITAÚ UNIBANCO S.A. 60.701.190/0001-04 - SISPAG via API (solicitar doc para canais de atendimento)
- Stark Bank S.A. 20.018.183/0001-80 - https://starkbank.com/docs/api#transfer https://starkbank.com/docs/api#dict
- Transfeera Serviços de Pagamentos Ltda. 27.084.098/0001-69 - https://docs.transfeera.dev/pagamentos/transferencias

Não PSPs, ou futuros PSPs em fase cadastral, homologação ou pré-operação (Empresa - Site ou Link API)

- We Pay Out Serviços de Pagamentos Ltda. 32.708.748/0001-30 - https://docs.wepayout.co/payout.html

**Comment by acgarzon** [comment]
BS2 permite o envio de PIX via API também

Manual: https://devs.bs2.com/manual/pix-clientes/

**Reply by MercadoSys** [reply]
A **PIX2PAY** Pagamentos é especialista em PIX e possui API também!

Link: https://www.pix2pay.com.br

**Reply by sostenesapollo** [reply]
Já vi aqui que parece demais com golpe, não entrem nesse

**Comment by leolima77** [comment]
BTG Pactual tem também! 

https://developer.btgpactual.com/docpm
https://developer.btgpactual.com/docpi

**Reply by felipe4labsit** [reply]
Safe2Pay: https://safe2pay.com.br
Ótimas tarifas

**Reply by rubenskuhl** [reply]
> Safe2Pay: https://safe2pay.com.br Ótimas tarifas

Em https://developers.safe2pay.com.br/ eu só localizei recebimento via Pix e envio de TED... há alguma doc específica de envio de Pix ?

**Reply by acgarzon** [reply]
> BTG Pactual tem também!
> 
> https://developer.btgpactual.com/docpm https://developer.btgpactual.com/docpi

Como você fez para ter acesso a API?

Eu abri a conta, mas não consigo acesso a API, aparentemente tem que se cadastrar somente para a API deles em um outro sistema.

Você falou com o seu gerente para ter acesso, ou algo do tipo?

**Reply by felipe4labsit** [reply]
> > Safe2Pay: https://safe2pay.com.br Ótimas tarifas
> 
> Em https://developers.safe2pay.com.br/ eu só localizei recebimento via Pix e envio de TED... há alguma doc específica de envio de Pix ?

@rubenskuhl Eles possuem cobrança e pagamento via PIX.

Para cobrança (recebimento), basta especificar a forma de pagamento PIX.
Veja:
https://developers.safe2pay.com.br/references/Invoice/invoice_create

E para envio de pagamento com valor fixo:
https://developers.safe2pay.com.br/references/Pix/payment_dynamic_pix

**Reply by rubenskuhl** [reply]
> > > Safe2Pay: https://safe2pay.com.br Ótimas tarifas
> > 
> > 
> > Em https://developers.safe2pay.com.br/ eu só localizei recebimento via Pix e envio de TED... há alguma doc específica de envio de Pix ?
> 
> @rubenskuhl Eles possuem cobrança e pagamento via PIX.
> 
> Para cobrança (recebimento), basta especificar a forma de pagamento PIX. Veja: https://developers.safe2pay.com.br/references/Invoice/invoice_create
> 
> E para envio de pagamento com valor fixo: https://developers.safe2pay.com.br/references/Pix/payment_dynamic_pix

No link que você colocou como de pagamento, diz que gera um QR-Code... o que seria um recebimento.

**Reply by rubenskuhl** [reply]
> Você falou com o seu gerente para ter acesso, ou algo do tipo?

@leolima77

**Reply by acgarzon** [reply]
> Safe2Pay: https://safe2pay.com.br Ótimas tarifas

Entrei em contato com o Safe2Pay e eles apenas oferecem o RECEBIMENTO de pix. Não efetuam o envio por API.

**Reply by leolima77** [reply]
É um dpto especifico dentro do BTG. Procura por alguém do BaaS (Banking as a Service), eles vão te orientar.

**Comment by moys2018** [comment]
Ola Boa Tarde.

Estou desenvolvendo QR Code Pix Dinâmico para ERP da Soft House em que trabalho, e gostaria de saber se alguém sabe se a Caixa Econômica Federal já se pronunciou em relação a API PIX ? 

Atenciosamente,
Obg.

**Reply by notrevetecnologia** [reply]
https://icobrei.com tem API PIX,PIX COBRANÇA & BOLETO

**Comment by rubenskuhl** [comment]
> https://icobrei.com tem API PIX,PIX COBRANÇA & BOLETO

Não é PSP participante do Pix, para boleto não é participante da COMPE, API Pix não segue padrão do Banco Central... parece o tipo de "facilitador" que o Banco Central está perto de regulamentar (e ao fazer isso, causar a extinção de boa parte deles). 

Além disso, este tópico é sobre envio de Pix, que não parece estar disponível nesse prestador.

**Comment by ApoloIslaio** [comment]


**Reply by rubenskuhl** [reply]
Mas se você vai ofertar contas transacionais, precisa mesmo de uma solução de contas, ou estará em violação do regramento do sistema financeiro...

**Reply by ApoloIslaio** [reply]
É né. Conhece alguma outra api com subconta? Acho que vou usar a iugu mesmo.

**Reply by rubenskuhl** [reply]
BTG e Rendimento.

**Reply by ApoloIslaio** [reply]
Acho esta api não permite criação de conta PF, só PJ.

**Reply by renatofrota** [reply]
Efí também tem subcontas. Tutorial [aqui](https://www.youtube.com/watch?v=uW78tu1xnSQ).

**Reply by ApoloIslaio** [reply]
Conheço, mais não é white label.

**Reply by notrevetecnologia** [reply]
Tem a iCobrei Pagamentos, precisar entrar em contato com eles, mas ele oferecem serviços, pra criação de contas PF e PJ, mas somente para serviços de CASHIN e CASHOUT PIX.

**Reply by ThalesFidelix** [reply]
> Tem a iCobrei Pagamentos, precisar entrar em contato com eles, mas ele oferecem serviços, pra criação de contas PF e PJ, mas somente para serviços de CASHIN e CASHOUT PIX.

OFF

**Comment by rubenskuhl** [comment]
> Tem a iCobrei Pagamentos, precisar entrar em contato com eles, mas ele oferecem serviços, pra criação de contas PF e PJ, mas somente para serviços de CASHIN e CASHOUT PIX.

Documentação da API da iCobrei só tem CashIn, não tem CashOut e não tem criação de contas...

**Comment by guibranco** [comment]
@rubenskuhl ICobrei foi retirado do ar...

**Reply by rubenskuhl** [reply]
Curioso, já que a demora da formalização da regulamentação de Gestor de Pagamentos tem permitido soluções "exóticas" continuarem operando.

**Comment by Clickpto1** [comment]
Tem também o Celcoin, eles tem uma documentação api muito boa.
https://developers.celcoin.com.br/docs/bem-vindo-%C3%A0-api-da-rede-celcoin

**Reply by ApoloIslaio** [reply]
já tinha visto eles, mas para utilizar a api de pix deles é R$ 2mil por mês.

**Reply by Clickpto1** [reply]
Pensando só em preço recomendo a do banco inter, zero taxa

**Reply by rubenskuhl** [reply]
Inter deixou de ser taxa zero. Era quando você postou, de fato.

**Reply by ThalesFidelix** [reply]
Mais cash-out ainda é taxa zero né? Só é difícil sacar a documentação direito, parece que é automatizado, mas não é...

**Comment by ApoloIslaio** [comment]


**Reply by Clickpto1** [reply]
Cheguei já ver eles, mais pelo oque entendi eles fornece o software, então para fazer um pix acaba pagando uma tarifa para eles e para o banco.

**Comment by rubenskuhl** [comment]
> Galera achei a api da OpenPix, pra mim é a melhor. O porém é que a criação de contas de usuários não é feita por eles, eu tenho que fazer o sistema que vai localizar o dinheiro do usuário. Mas a questão de regulamentação com o Bacen, fica com eles. É como se fosse uma conta bolsão da empresa que fica na OpenPix e então oferecemos uma conta transacional para os usuários. Pelo preço, sem dúvidas é a melhor.
> 
> https://openpix.com.br/

O que você está propondo de conta bolsão não é o que eles fazem, é como você está querendo estruturar... e isso contraria um bom número de regulamentos do Banco Central. 

Há soluções de BaaS como Flagship(Matera), BTG e Celcoin aonde os titulares de contas efetivamente detém uma conta numa instituição de pagamento autorizada.

**Reply by ApoloIslaio** [reply]
OpenPix é autorizada.

**Reply by ApoloIslaio** [reply]
Termos uma conta na OpenPix cnpj e então ofertar uma carteira para usuários pf não seria possivel em questão de regulamentação?

**Reply by rubenskuhl** [reply]
Sim, isso é exercer atividade regulada sem ter as credenciais devidas. E independe de aonde você tem conta (que não é na OpenPix, você provavelmente tem conta Flagship que é a usada para maior parte das atividades da OpenPix).

**Comment by rubenskuhl** [comment]
> OpenPix é autorizada.

Não é autorizada e não se diz autorizada. Procure a empresa em https://www.bcb.gov.br/estabilidadefinanceira/participantespix . 
Eles tem planos de se tornar, mas ainda não aparecem nem como empresa em processo de se tornar.

**Reply by ApoloIslaio** [reply]
a https://woovi.com/  utiliza a mesma api. Essas duas empresas utilizam a api dessa  Flagship ?

**Comment by rubenskuhl** [comment]
> Fiz call com eles, e disseram que eu poderia usar a api dessa forma e que a regulamentação com Bacen era com eles.



> a https://woovi.com/ utiliza a mesma api. Essas duas empresas utilizam a api dessa Flagship ?

Woovi e OpenPix são a mesma empresa. Eles tem integrações com vários bancos, mas para quando você não passa para ele uma conta sua em banco que eles suportem, eles usam Flagship/Matera. É o que aparece inclusive quando alguém faz um Pix, que a conta é na Flagship. Não é segredo essa relação.

**Reply by ApoloIslaio** [reply]
Vi que a Flagship é autorizada.

**Comment by rubenskuhl** [comment]
> Vi que a Flagship é autorizada.

Autorizada a deter contas no sistema financeiro... mas para maior clareza, não é autorizada a autorizar terceiros. Só quem tem poder e dever legal para isso é o Banco Central.

**Comment by ggarlera** [comment]
Olá... gostaria de saber se alguém tem uma tabela de retorno de erros do PIX ? Se possui algum padrão já definido do BC ou se no arranjo já possuem algo?

**Comment by Coffegun** [comment]
Olá pessoal,

Estou precisando de um ajuda, estou criando um disparador para fazer o pagamento via PIX no WhatsApp, e gostaria de saber se é realmente obrigatório ter que passar o CPF ou CNPJ do pagador do PIX para recebedores PSP. Na documentação não deixa claro se de fato é obrigatório. Poderiam me ajudar, por favor? 

Muito obrigado!

**Reply by rubenskuhl** [reply]
É opcional mas aí não pode especificar nem nome nem CPF, tem que omitir a árvore devedor

**Comment by lucaspercisi** [comment]
Banco do Brasil tem também: https://apoio.developers.bb.com.br/referency
Bradesco vi algo nessa documentação: https://exclusive.bradesco/pix/assets/docs/api_pix_200.pdf
Sicredi possui algo entrando em contato com a agência: https://www.sicredi.com.br/site/pixpj/api-pix/
Santander: https://developer.santander.com.br/api/documentacao/pix#/
Sicoob: https://developers.sicoob.com.br/portal/

**Reply by acgarzon** [reply]
> Banco do Brasil tem também: https://apoio.developers.bb.com.br/referency Bradesco vi algo nessa documentação: https://exclusive.bradesco/pix/assets/docs/api_pix_200.pdf Sicredi possui algo entrando em contato com a agência: https://www.sicredi.com.br/site/pixpj/api-pix/ Santander: https://developer.santander.com.br/api/documentacao/pix#/ Sicoob: https://developers.sicoob.com.br/portal/

BB tem envio PIX (cashout)?

Todos estes que você listou tem pix cashout?

**Comment by jadersonk** [comment]
Pessoal, boa noite.

Eu preciso de um apoio de vocês: estou estruturando operações de receitas (recebimento) e pagamentos por API para integrar em um negócio, e depois de uma exaustiva pesquisa de Mercado e Documentações regulatórias, fiquei muito preocupado com o grau de Risco atribuído aos índices de reclamações do Reclame Aqui e outras Instituições de pagamentos, por questões trivialmente comum. Se vocês pesquisarem a fundo verão que por efeito, as mudanças fiscais e operacionais do BCB, Estão impactando diretamente na manutenção destes tipos de serviços ofertados às contas digital banking, por exigências regulatórias - vi casos de valores absurdos retidos por Instituições de Pagamentos em desfavor de titular de negócio, em virtude de conta bloqueada por exigência de conformidade com BCB, de uns 3 meses pra cá de reclamações. E até que se forneça, toda a papelada, o operacional do negócio fica travado! 

Então diante disto, quais Instituições de pagamento vocês me indicam como mais "safe", "e já desenvolveram para", para que eu possa integrar as operações de PIX em uma infraestrutura de negócio via API? Considerando uma empresa contratante 100% em dia com suas obrigações, na função de APTA junto a RFB. Obrigado!

**Reply by rubenskuhl** [reply]
Eu não tenho essa leitura do regulatório... e o que você relata é chamado de "bloqueio cautelar", mas que pelo regulamento só se aplica a contas CPF, não a contas CNPJ. 

APTA junto à RFB só quer dizer que está declarando impostos... uma operação criminal pode declarar impostos corretamente, e tanto o regulatório quanto as leis requerem um pouco mais do que isso. E muitos termos de uso de IFs/IPs especificam restrições adicionais mesmo a atividades lícitas mas que são tidos de risco por aquela casa, por exemplo dropshipping, betting etc. 

O que eu sugiro é ter bastante aos termos de uso antes de escolher o seu PSP. Tudo que um PSP não gosta, já está descrito lá... o resto, não sendo ilegal/lavagem de dinheiro/financiamento de terrorismo, é legítimo. 

Os PSPs recebedores estão listados no issue #76 , este issue é de PSPs para envio de Pix. Entre os listados nesse issue, o que usamos é a Efí e temos um bom histórico de bom-senso da parte deles em analisar MEDs (O "chargeback" do Pix) e RDRs (reclamações de consumidores abertas diretamente no BACEN) 100% a nosso favor. Claro que ajuda nisso fazermos tudo certo, mas o mesmo processo no arranjo de cartões frequentemente tem decisões desfavoráveis , mostrando que o arranjo Pix é bem mais robusto nessa questão.

**Reply by jadersonk** [reply]
Agradeço pela resposta, estarei analisando a issue #76 atentamente. Mas minha intenção foi justamente obter uma **"recomendação"** de gateways de pagamentos como citatestes, com base em práticas de aderência em nas próprias infraestruturas dos colegas desenvolvedores que já realizaram integrações. Pois como atuo há 19 anos no mercado financeiro de compliance, tendo passagem inclusive em Bancos e Cia de seguros (compliance) nos papéis de liderança e desenvolvimento, tenho know how acerca do assunto que envolve Mercado Regulatório, Processos Internos, BCB e CVM, Compliance, KYC, AML, Atas ADs, NRs, entre outros adotando tais práticas para estruturação de negócio homologável. E isto é realmente exaustivo. Enfim, aquelas operações de negócios por convenção restritas a área de risco e compliance, seja por exigência Institucional ou necessidade organizacional.

Apenas destaquei os "motivos" da minha procura, visto que estou observando situações "atípicas" de ações que me acenderam o alerta, para um vetor de clientes não qualificados por uma característica, o que me fez reconsiderar a continuidade com um PSP. O BACEN vem realizando inovações frequentes desde 2023 num processo de aceleração, e creio que de alguma forma isto está afetando a carteira de cliente motivado pelo impacto nos Processos Internos dos Provedores de Pagamentos.

Então deixo o meu alerta aos colegas, para que escolham e monitorem as Instituições e Provedores de Pagamentos PSPs integrados as suas Infraestruturas de operações por API. identifiquei reclamações de clientes, tendo sua infraestrutura operacional de negócio congelada, por retenção de capital de giro de Pessoa Jurídica "sem aviso prévio" nas Contas vinculantes aos PSPs, mesmo estando com alinhamento fiscal na Receita Federal, e CNAE coerente - sem nenhuma investigação em andamento. Casos estes que já ingressaram com ações na justiça nos últimos dias. 
Apenas não vou citar nomes por questões de imagem, mas já foram citados nesta conversa.

Obrigado!

**Reply by rubenskuhl** [reply]
Se as ações na Justiça não foram abertas pedindo sigilo, basta citar os números dos processos...

**Comment by alyssoncrvg** [comment]
essa lista está disponível apenas para contas pj? quero fazer essa integracao porém em uma conta pf, alguém poderia me tirar essa dúvida?

**Reply by rubenskuhl** [reply]
> essa lista está disponível apenas para contas pj? quero fazer essa integracao porém em uma conta pf, alguém poderia me tirar essa dúvida?

Na API de envio de Pix, o problema é que o regulamento do Pix não tem uma abertura clara para permitir cobrança de tarifas de PF nessa transação. Então por conservadorismo, as que tenho mais contato não permitem envio de Pix via API por PF. 

Mas essa não é uma característica mapeada; pode ser que haja alguma exceção. Mas é um requisito difícil mesmo.

**Comment by rubenskuhl** [comment]
Só para comentar que como na IN 514 o Banco Central proibiu repassar dados de conta armazenados no DICT para terceiros (vide https://www.bcb.gov.br/estabilidadefinanceira/exibenormativo?tipo=Instru%C3%A7%C3%A3o%20Normativa%20BCB&numero=514 ) , APIs de envio de Pix não podem mais desde o dia 02.09.2024 responder os dados e depois deixar uma nova requisição ser feita para os dados manuais.

**Comment by notrevetecnologia** [comment]
ICobrei Foi comprada pela NOTREVE INSTITUIÇÃO DE PAGAMENTO(
https://notrevepay.com.br) e sim eles fazem CASHIN e CASHOUT, abertura de
contas e etc.

Em sáb., 15 de mar. de 2025, 11:48, ThalesFidelix ***@***.***>
escreveu:

**Comment by notrevetecnologia** [comment]
Eu uso, mas tem taxa. Eu pago 0,30 de Pix CASHIN e CASHOUT mas as taxas
eles negócia de acordo com volume

Em sáb., 15 de mar. de 2025, 12:00, ThalesFidelix ***@***.***>
escreveu:

**Reply by ThalesFidelix** [reply]
Curioso, fui estudar, mas....infelizmente insta fora do ar e site em manutenção

**Comment by notrevetecnologia** [comment]
Golpe? Não faz sentido essa afirmação.
Eu utilizo a plataforma desde a época em que era iCobrei, e acompanhei toda
a evolução. Eles foram adquiridos e hoje operam como subadquirente da
Celcoin, que é uma fintech séria e regulamentada.
O serviço mudou, melhorou e continua funcionando perfeitamente.
Eu uso, confio e recomendo.

Em dom., 27 de abr. de 2025, 10:37, Sóstenes Apollo <
***@***.***> escreveu:

**Comment by jlmachado** [comment]
P2Ps que oferecem por meio de PF , no meu caso terminei o sistema e não dá para testar gastando muito

**Reply by guibranco** [reply]
Aqui é lista de PSP, não P2P, do SPI.

O que você está procurando é P2P de criptomoeda? Ou o que exatamente?

**Comment by albertobraschi** [comment]
> Banco do Brasil tem também: https://apoio.developers.bb.com.br/referency
> 
> Bradesco vi algo nessa documentação: https://exclusive.bradesco/pix/assets/docs/api_pix_200.pdf
> 
> Sicredi possui algo entrando em contato com a agência: https://www.sicredi.com.br/site/pixpj/api-pix/
> 
> Santander: https://developer.santander.com.br/api/documentacao/pix#/
> 
> Sicoob: https://developers.sicoob.com.br/portal/

Sicredi so libera api pix para correntista

**Reply by rubenskuhl** [reply]
> > Banco do Brasil tem também: https://apoio.developers.bb.com.br/referency
> > Bradesco vi algo nessa documentação: https://exclusive.bradesco/pix/assets/docs/api_pix_200.pdf
> > Sicredi possui algo entrando em contato com a agência: https://www.sicredi.com.br/site/pixpj/api-pix/
> > Santander: https://developer.santander.com.br/api/documentacao/pix#/
> > Sicoob: https://developers.sicoob.com.br/portal/
> 
> Sicredi so libera api pix para correntista

API Pix pressupõe uso exclusivo por correntista mesmo, em qualquer PSP.

**Comment by notrevetecnologia** [comment]
Abre PF e PJ, se negóciar conseque taxas de até 0,30.
https://notrevepay.com.br

Em qua., 11 de jun. de 2025, 09:00, jlmachado ***@***.***>
escreveu:

**Comment by pedropedruzzi** [comment]
MercadoPago oferece: https://www.mercadopago.com.br/developers/pt/docs/money-out/landing
